### DISCUSSION TITLE
GODADDY域名更新失败，wsarecv: An existing connection was forcibly closed by the remote host

### DISCUSSION THREAD

**Discussion by vxlinjie** [jeessy2/ddns-go#1018] [Q&A]
专门买了个godaddy域名来做ddns，结果失败了，

2024/02/16 16:59:06 更新域名解析 1.lilinjie.online 失败! 异常信息: Put "https://api.godaddy.com/v1/domains/lilinjie.online/records/A/1": read tcp 192.168.6.164:63046->23.194.70.78:443: wsarecv: An existing connection was forcibly closed by the remote host.

唉，估计是墙的原因。后来换了阿里云域名就正常了。

**Comment by yin1999** [comment]
要使用godaddy来做DDNS的话建议配置代理，程序支持使用 `HTTPS_PROXY`/`HTTP_PROXY` 环境变量来让 API 请求通过代理进行转发
